diff --git a/tools/mapjson/mapjson.cpp b/tools/mapjson/mapjson.cpp
index dd6fbcf3..16b28da0 100644
--- a/tools/mapjson/mapjson.cpp
+++ b/tools/mapjson/mapjson.cpp
@@ -494,16 +494,21 @@ string generate_groups_text(Json groups_data, vector<string> &invalid_maps) {
     for (auto &key : groups_data["group_order"].array_items()) {
         string group = json_to_string(key);
         vector<string> valid_maps;
+        size_t real_maps = 0;
         auto maps = groups_data[group].array_items();
         for (Json &map_name : maps) {
             string map_name_str = json_to_string(map_name);
             auto it = find(invalid_maps.begin(), invalid_maps.end(), map_name_str);
             if (it == invalid_maps.end()) {
                 valid_maps.push_back(map_name_str);
+                real_maps++;
+            } else {
+                // keep the slot so later maps keep their MAP_* numbers
+                valid_maps.push_back("NULL");
             }
         }
 
-        if (valid_maps.size() > 0) {
+        if (real_maps > 0) {
             text << group << "::\n";
             for (string map : valid_maps)
                 text << "\t.4byte " << map << "\n";
